docs/integrations/destinations/README.md

```diff
@@ -4,3 +4,76 @@ description: 'Data warehouses, data lakes, databases...'
 
 # Destinations
 
+
+
+## BigQuery Naming
```

Comment:
Shouldn't that documentation be in the docs for each of the destination?

Reply:
i can dispatch them yes, I just started in the same file as it's easier to see how different or similar each one of them were